php/src/Google/Protobuf/Internal/AnyBase.php

```diff
@@ -79,6 +79,9 @@ public function is($klass)
     {
         $pool = \Google\Protobuf\Internal\DescriptorPool::getGeneratedPool();
         $desc = $pool->getDescriptorByClassName($klass);
+        if (is_null($desc)) {
+            return false;
```

Comment:
Log "The class is not a proto message or hasn't been instantiated yet. Make sure to instantiate this proto class first." and throw error.